### PR TITLE
Bugfix/ascii satwind datetime fix for SPRINT

### LINKED ISSUE #925: bug fix for datetime in SSEC satellite wind decoder
## Description

the satellite wind decoder for SSEC in the ioda-converter-sprint branch needs an update to the datetime decoding

## Requirements

the fix of this bug will correctly assign the time and allow observation to process in UFO

## Acceptance Criteria (Definition of Done)

IODA satwind file gets created

### PR DESCRIPTION
**Pull request by BenjaminRuston**
## Description

for the ioda-converter SPRINT branch

correct the extraction of the date and time from the SSEC ascii geostationary satellite winds

### Issue(s) addressed

- fixes #925 

## Acceptance Criteria (Definition of Done)

successful conversion of data and testing through UFO

### CODE REVIEW

File: src/conventional/amv_ssec_ascii2ioda.py [hunk 1]
```diff
@@ -187,17 +187,30 @@ def read_file(file_name, data):
...
-                month = int(row['day'][5:6])
-                day = int(row['day'][7:8])
-                hour = int(row['hms'][0:1])
-                minute = int(row['hms'][2:3])
+                month = int(row['day'][4:6])
+                day = int(row['day'][6:])
+                hour = int(row['hms'][0:2])
+                minute = int(row['hms'][2:])
                 second = 0
                 dtg = datetime(year, month, day, hour, minute, second)
```
Comment (fabiolrdiniz) [inline src/conventional/amv_ssec_ascii2ioda.py:195]:
```suggestion
                dtg = datetime.strptime(f"{row['day']} {row['hms']}", '%Y%m%d %H%M')
```

**Review by fabiolrdiniz — APPROVED** [review]
Hi, @BenjaminRuston. Thanks for adding me here. I would suggest a simplification for the handling of dates, but I don't know if that is acceptable here.

File: src/conventional/amv_ssec_ascii2ioda.py (hunk 1 above)
Comment (BenjaminRuston) [reply src/conventional/amv_ssec_ascii2ioda.py:195]: yup I like this!

**Review by gthompsnJCSDA — APPROVED** [review]
Oh, I like Fabio's suggestion a LOT.  I forgot how nice the formatting can be done with the datetime object in python.

**Review by haydenlj — APPROVED** [review]
Agreed, that's a good suggestion. Much simpler

**Comment by srherbener** [comment]
Relevant tests are passing so I will merge now.

**Pull request merged**